packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -25,48 +25,68 @@ const propTypes = {
    */
   accessoryAlignment: PropTypes.oneOf(['alignTop', 'alignCenter']),
   /**
+   * The height and width value the accessory should receive. Maximum scalar is 60px.
+   */
+  accessoryScale: PropTypes.string,
+  /**
    * The react element to be placed in the start aligned accessory position.
    */
-  startAccessory: PropTypes.element,
+  startAccessory: PropTypes.node,
+  /**
+   * Indicates whether or not space is allocated for the start accessory if none is given.
+   */
+  alignStartAccessory: PropTypes.bool,
   /**
    * The react element to be placed in the end aligned accessory position.
    */
-  endAccessory: PropTypes.element,
+  endAccessory: PropTypes.node,
+  /**
+   * Indicates whether or not space is allocated for the end accessory if none is given.
+   */
+  alignEndAccessory: PropTypes.bool,
```

Comment:
It might be late in the game for a change but these prop names match the props in Arrange but mean completely different things which is a little confusing. 

https://github.com/cerner/terra-core/blob/master/packages/terra-arrange/src/Arrange.jsx#L26-L34

The name implies the prop is performing some alignment to the start / end accessory rather than preserving or providing default spacing

Reply:
That is a good point. Originally I had thought something like `hasStartAccessorySpacing` but that was long & ugly. Any naming ideas?